### DISCUSSION TITLE
DS2415+ 2.5GB is this possible.  CPU type does not seem to be listed in the releases section.

### DISCUSSION THREAD

**Discussion by Stingeremu** [bb-qq/r8152#456] [Q&A] [answered]
I have an INTEL Atom C2538 - Products formerly Rangeley CPU and I'm unable to find a driver for my CPU type.

I'm assuming that running a 2.5GB adaptor is a non starter on the DS2415+ but just wanted to check with someone in the hope there may be another driver type I can use.

If not I guess I need to upgrade.  I've had the NAS for 10 years so it's served me very well, just hoping to get a little more life out of it.

Thanks

**Comment by JohnVillalovos** [comment, marked as answer]
I think it falls under Avoton. I have a DS415+ with the same CPU. And on the compatibility page[1] in the Wiki people are calling it Avoton.

Also at Wikipedia https://en.wikipedia.org/wiki/List_of_Intel_Atom_processors  The Avoton and Rangeley are both Silvermont microarchitecture.


[1] https://github.com/bb-qq/r8152/wiki/Compatibility

**Reply by Stingeremu** [reply]
Thanks for coming back to me John, Really appreciate it.  Have you used the driver and had success with it? I think it'll purchase a new switch and the USB to LAN adaptor today and give it a go.  

Thanks Again

**Reply by JohnVillalovos** [reply]
I bought https://www.amazon.com/dp/B0C4Y5GWC7 and https://www.amazon.com/dp/B0CD1FDKT1. And connected it to the back USB port on my DS415+. So far it seems to be working, but I have only tested at 1 Gbit.

I am still in the process of setting things up and getting my PC also going at 2.5Gb. I'm waiting on a 2.5Gb PCIe network card to arrive. For my switch I got the Unifi Flex Mini 2.5G, https://store.ui.com/us/en/category/switching-utility/products/usw-flex-2-5g-5 but I would only recommend that if you are using Unifi or want to get into the Unifi ecosystem.

**Reply by Stingeremu** [reply]
Just wanted to update you.  Finally got around to giving this ago.  I bought the USB to Nic adapter you suggested and went with a MonkerLink 8 Port 2.5G Switch (nothing to fancy) cost just under £50 from amazon.

Worked like a charm 2.5gb Woo Hoo

There's life in the old DS2515 yet!!

All the best for 2025 and thanks again for you help

**Reply by JohnVillalovos** [reply]
Great news. Glad it is working. I usually get around 200-220 MBytes/second on my transfers to/from the DS415+ and my Windows machine. So I am happy. I get slightly slower speeds to my Linux box but that might be because of disk encryption and other things.

**Comment by bb-qq** [comment]
The problem seems to be resolved, so this conversation is closed. If you have any other questions, feel free to re-open.
